certbot-nginx/certbot_nginx/parser.py

```diff
@@ -104,9 +104,10 @@ def _build_addr_to_ssl(self):
 
     def _get_raw_servers(self):
         # pylint: disable=cell-var-from-loop
+        # type: () -> Dict
         """Get a map of unparsed all server blocks
         """
-        servers = {}
+        servers = {} # type: Dict
```

Comment:
`Dict[str, Union[List, nginxparser.UnspacedList]]`

@jameshiebert, I'd use the type in my comment for now, but I'm curious what you think @ohemorange.

Reply:
yes, that's correct. hopefully with sydney's changes we can update this to not need a union.